graph/GuidelinesGraph.md

```diff
@@ -330,6 +319,136 @@ For a complete mapping of error codes to HTTP statuses, see
 
 <a name="api-contract-and-non-backward-compatible-changes"></a>
 
+### Enums
+
+In OData, enums represent a subset of the nominal type they rely on, and are especially useful in cases where certain properties have predefined, limited options.
+
+```xml
+<EnumType Name="Color">
+    <Member Name="Red" Value="0" />
+    <Member Name="Green" Value="1" />
+    <Member Name="Blue" Value="2" />
+</EnumType>
+```
+
+#### Pros
+
+- Our SDK generators will translate the enum to the best representation of the target programming language, resulting in a better developer experience and free client side validation
+
+#### Cons
+
+- Adding a new value requires to go through a (generally fast) API Review
+- If the enum is not [evolvable](./patterns/evolvable-enums.md), adding a new value is a breaking change and will generally not be allowed
+
+#### Enum or Booleans
+
+Enumerations are a good alternative to Booleans when one of the two values (`true`, `false`) conveys other possible values not yet conceived. Let's assume we have an `Error` type and a property to communicate how to display it:
+
+```xml
+<ComplexType Name="Error">
```

Comment:
NIT: all type names should be camel case. Also, we have a ComplexType "publicError" that has dependencies on a utility library. I would take a different real example to avoid creating confusion for our customers.

Reply:
Yes. Please create a different example or, better yet, find a good existing one. Also, since we do not allow new non-evolvable enums, why don't you add these considerations to the existing pattern?

Reply:
I wouldn't say that all Enums are evolvable. For example, Day of Week wouldn't be. Also, it's up to the API producer to choose whether to use an Enum or another construct for their use cases.